helm/dev-values.yaml

```diff
@@ -1,6 +1,5 @@
 environment: development
 container:
-  redisname: redis-redis-ha.mine-support
```

Comment:
Is this part of this change or has it snook in?

Reply:
The dev `redisname` has changed to be the same as the default in `values.yaml`, so the override is no longer needed.